### DISCUSSION TITLE
Bas de page

### DISCUSSION THREAD

**Discussion by maximpouliot** [MTESSDev/FRW#115] [Annonces]
Bonjour, est-ce que le bas de page (autre que le logo) est personnalisable? 
Notre experte UX nous dit : 
Selon les normes PIV, la mention doit être «© Gouvernement du Québec, [date de première publication]» 

Merci!

**Comment by anisite** [comment]
Bonjour, donc il nous manquerait seulement la virgule ?
![image](https://github.com/MTESSDev/FRW/assets/3194288/57903e27-3294-4024-969d-265cf4fdeb2a)

Par contre, l'outil été publié en 2021, la date qui y est inscrite est celle de FRW, je ne vois pas trop ce que vous voudriez ajouter, mais nous pouvons y mettre la virgule sans problème.

Merci de nous informer

**Reply by jodionrq** [reply]
Bonjour,
Oui il s'agirait simplement d'ajouter la virgule pour être conforme avec notre PIV.
Donc, si c'est vous qui faite le changement je vous remercie à l'avance.
Si nous avons un endroit pour changer ce paramètre nous-même, merci de nous l'indiquer.
Bonne journée!

**Comment by j0ber** [comment]
Bonjour,
Il y avait effectivement une erreur dans notre fichier de config default.
La virgule était manquante dans le copyright.
Nous avons corrigé le tout et c'est déjà effectif.

Désolé pour le délai, période des vacances...

Merci de nous l'avoir signalé!

**Reply by jodionrq** [reply]
Merci beaucoup pour le suivi!
Bonne journée.
